code/modules/mob/new_player/sprite_accessories_tail.dm

```diff
@@ -238,6 +267,57 @@
 	extra_overlay = "seromitail_feathers_s"
 	species_allowed = list(SPECIES_TESHARI, SPECIES_EVENT1, SPECIES_EVENT2, SPECIES_EVENT3)
 
+/datum/sprite_accessory/tail/special/tesharitail_noplume
+	name = "teshari tail, no plumage"
+	icon_state = "teshtail_noplume_s"
+	species_allowed = list(SPECIES_TESHARI, SPECIES_EVENT1, SPECIES_EVENT2, SPECIES_EVENT3)
+	color_blend_mode = ICON_ADD
+
+/datum/sprite_accessory/tail/special/tesharitail_noplume_feathered
+	name = "teshari tail, no plumage w/feathers"
+	icon_state = "teshtail_noplume_s"
+	extra_overlay = "teshtail_noplume_feathers_s"
+	species_allowed = list(SPECIES_TESHARI, SPECIES_EVENT1, SPECIES_EVENT2, SPECIES_EVENT3)
+	color_blend_mode = ICON_ADD
+
+/datum/sprite_accessory/tail/special/tesharitail_damaged
+	name = "teshari tail, damaged"
+	icon_state = "teshtail_damaged_s"
+	species_allowed = list(SPECIES_TESHARI, SPECIES_EVENT1, SPECIES_EVENT2, SPECIES_EVENT3)
+	color_blend_mode = ICON_ADD
+
+/datum/sprite_accessory/tail/special/tesharitail_damaged_feathered
+	name = "teshari tail, damaged w/feathers"
+	icon_state = "teshtail_damaged_s"
+	extra_overlay = "teshtail_damaged_feathers_s"
+	species_allowed = list(SPECIES_TESHARI, SPECIES_EVENT1, SPECIES_EVENT2, SPECIES_EVENT3)
+	color_blend_mode = ICON_ADD
+
+/datum/sprite_accessory/tail/special/tesharitail_stubby
+	name = "teshari tail, stubby"
+	icon_state = "teshtail_stubby_s"
+	species_allowed = list(SPECIES_TESHARI, SPECIES_EVENT1, SPECIES_EVENT2, SPECIES_EVENT3)
+	color_blend_mode = ICON_ADD
+
+/datum/sprite_accessory/tail/special/tesharitail_stubby_feathered
+	name = "teshari tail, stubby w/feathers"
+	icon_state = "teshtail_stubby_s"
+	extra_overlay = "teshtail_stubby_feathers_s"
+	species_allowed = list(SPECIES_TESHARI, SPECIES_EVENT1, SPECIES_EVENT2, SPECIES_EVENT3)
+	color_blend_mode = ICON_ADD
+
+/datum/sprite_accessory/tail/special/tesharitail_prosthetic_unbranded
+	name = "teshari tail, unbranded prosthetic"
+	icon_state = "teshtail_prosthetic_unbranded_s"
+	species_allowed = list(SPECIES_TESHARI, SPECIES_EVENT1, SPECIES_EVENT2, SPECIES_EVENT3)
+	do_colouration = FALSE
+
+/datum/sprite_accessory/tail/special/tesharitail_prosthetic_cenilimi
+	name = "teshari tail, Cenilimi Cybernetic prosthetic"
```

Comment:
"Cenilimi Cybernetic**s** prosthetic" since it's a brand name, or Cenilmini cybernetic prosthetic if you're using a shortened version of the name and cybernetic is an adjective. (I'd go for the first one for consistency with other prosthetic bits personally)

Otherwise all good from my end lore-wise. Variety is good, and most of these are realistically disfigurements and such so nothing controversial really. Prosthetic tail is important for balance!

Reply:
Change made!